lib/Basics/files.cpp

```diff
@@ -276,16 +299,37 @@ bool TRI_IsSymbolicLink(char const* path) {
 /// @brief creates a symbolic link
 ////////////////////////////////////////////////////////////////////////////////
 
-bool TRI_CreateSymbolicLink(std::string const& target, std::string const& linkpath, 
-                            std::string& error) {
+bool TRI_CreateSymbolicLink(std::string const& target,
+                            std::string const& linkpath, std::string& error) {
 #ifdef _WIN32
-  // TODO : check if a file is a symbolic link - without opening the file
-  return false;
```

Comment:
please use   TRI_ERRORBUF; && TRI_SYSTEM_ERROR(); for error handling as it is elsewhere.

Reply:
please fix the macro to use FormatMessageW + convert back to utf8